backend/src/api/integration/helpers/twitterAuthenticate.ts

```diff
@@ -9,8 +9,7 @@ export default async (req, res, next) => {
 
   const state = {
     tenantId: req.params.tenantId,
-    redirectUrl: req.query.redirectUrl,
-    hashtags: req.query.hashtags,
```

Comment:
I'd say let's keep the hashtags in the state instead of redirect url - so the fields required are separated 
The reason hashtags were not being forwarded back to authenticate callback is it's missing in `redisPKCEStore` line 49. After adding the hashtags there it should forward successfully